lib/ethpy/ethpy/eth_config.py

```diff
@@ -41,15 +48,15 @@ def build_eth_config() -> EthConfig:
     # Look for and load local config if it exists
     load_dotenv("eth.env")
 
-    artifacts_url = os.getenv("ARTIFACTS_URL")
-    rpc_url = os.getenv("RPC_URL")
+    artifacts_uri = os.getenv("ARTIFACTS_URI")
+    rpc_uri = os.getenv("RPC_URI")
     abi_dir = os.getenv("ABI_DIR")
 
     arg_dict = {}
-    if artifacts_url is not None:
-        arg_dict["ARTIFACTS_URL"] = artifacts_url
-    if rpc_url is not None:
-        arg_dict["RPC_URL"] = rpc_url
+    if artifacts_uri is not None:
+        arg_dict["ARTIFACTS_URL"] = artifacts_uri
```

Comment:
Bug here, arg_dict is setting this variable as `ARTIFACTS_URL`, which I found surprising that considering this PR changes the field for EthConfig to `artifacts_uri`

Reply:
Explicit testing says this should fail with unexpected keyword, so again, I have no idea how this is passing CI